src/buzz/buzzparser.c

```diff
@@ -650,6 +654,8 @@ int parse_for(buzzparser_t par) {
    if(!parse_expression(par)) return PARSE_ERROR;
    tokmatch(BUZZTOK_PARCLOSE);
    fetchtok();
+   // Allow the use of non-cuddled brackets
```

Comment:
Please avoid the C++ comment style. Use `/* comment */` instead of `// comment`.